ibm_db2/datadog_checks/ibm_db2/ibm_db2.py

```diff
@@ -7,19 +7,28 @@
 from time import time as timestamp
 
 import ibm_db
+import re
 
 from datadog_checks.base import AgentCheck, is_affirmative
 from datadog_checks.base.utils.containers import iter_unique
 
 from . import queries
-from .utils import scrub_connection_string, status_to_service_check
+from .utils import get_version, scrub_connection_string, status_to_service_check
 
 
 class IbmDb2Check(AgentCheck):
     METRIC_PREFIX = 'ibm_db2'
     SERVICE_CHECK_CONNECT = '{}.can_connect'.format(METRIC_PREFIX)
     SERVICE_CHECK_STATUS = '{}.status'.format(METRIC_PREFIX)
     EVENT_TABLE_SPACE_STATE = '{}.tablespace_state_change'.format(METRIC_PREFIX)
+    VERSION_REGEX = re.compile(
```

Comment:
Do you need a custom regex?
This matches 11.1.1 but not 10.1.1 or 12.1.1 :/

Reply:
I'll update the major regex. The `patch` rule fails for the `update/release` section of the ibm_db2. The example we get `11.1.0202` fails the semver regex because of the leading 0.

Reply:
Maybe just manipulate the string to make it semver

Reply:
So would the default transformer work?